README.md

```diff
@@ -31,7 +31,7 @@ We need to have all these sessions backed up somewhere if we do not want to lose
 
 The following are required before you can enable Hazelcast Session Clustering:
 
--   Target application or web server should support Java 1.6 or higher.
+-   Target application or web server should support Java 1.8 or higher.
```

Comment:
This actually depends on Hazelcast version
5.0 - 5.2 - 1.8
5.3 - 11
5.4 - 17+